src/providers/mssql/qgsmssqlprovider.cpp

```diff
@@ -2973,8 +2973,11 @@ bool QgsMssqlProvider::getExtentFromGeometryColumns( QgsRectangle &extent ) cons
   QSqlQuery query = createQuery();
   query.setForwardOnly( true );
 
-  QString sql = QStringLiteral( "SELECT qgis_xmin,qgis_xmax,qgis_ymin,qgis_ymax FROM geometry_columns WHERE f_table_name = '%1' AND NOT (qgis_xmin IS NULL OR qgis_xmax IS NULL OR qgis_ymin IS NULL OR qgis_ymax IS NULL)" );
-  QString statement = sql.arg( mTableName );
+  QString sql = QStringLiteral( "SELECT qgis_xmin,qgis_xmax,qgis_ymin,qgis_ymax "
+                                "FROM geometry_columns WHERE f_table_name = '%1' AND f_table_schema = '%2' "
+                                "AND NOT (qgis_xmin IS NULL OR qgis_xmax IS NULL OR qgis_ymin IS NULL OR qgis_ymax IS NULL)" );
+
+  QString statement = sql.arg( mTableName, mSchemaName );
```

Comment:
We should be escaping these table and schema names, right?

Reply:
sure...